docs/api.md

```diff
@@ -242,12 +239,10 @@ number of records rather than returning the records themselves.
 
 ### Parameters
 
-Name | Type | Required | Description
-:--: | :--: | :------:| -----------
-resource | String | Yes | The resource to query
-where | Object | No | Specifies constraints to filter the data.  Null means all records will be returned.
-
-The `where` is an object with supported operations defined in [API operations](https://dev.vantiq.com/docs/system/api/index.html#where-parameter).
+**`String resource`**:  (*Required*) The name of the resource type to query.
+ Must be either `SystemResources.TOPICS.value()`, `SystemResources.SOURCES.value()` or `SystemResources.TYPES.value()`.
```

Comment:
Will these references like "SystemResources.TOPICS.value()" will work in Node.js?
They look like Java code to me...